server/src/services/organizations.service.ts

```diff
@@ -708,10 +708,13 @@ export class OrganizationsService {
             );
           }
 
-          this.inviteUserswrapper(users, currentUser).catch((error) => {
-            console.error(error);
-          });
-          res.status(201).send({ message: `${rowCount} user${isPlural(users)} are being added` });
+          this.inviteUserswrapper(users, currentUser)
```

Comment:
`inviteUserswrapper` is called asynchronously. It takes time to process if number of users are more.

Reply:
@arpitnath will discuss with product team